src/clj/rems/migrations/example.clj

```diff
@@ -0,0 +1,9 @@
+(ns rems.migrations.example
```

Comment:
a word somewhere about how this is an example of how to do clj migrations would be nice

Reply:
Done.